validator/client/src/integration-test/java/tech/pegasys/teku/validator/client/signer/ExternalSignerIntegrationTest.java

```diff
@@ -119,6 +118,36 @@ void failsSigningWhenSigningServiceReturnsInvalidSignatureResponse() {
             "External signer returned an invalid signature: Illegal character 'I' found at index 0 in hex binary representation");
   }
 
+  @Test
+  void failsSigningBlockWhenSigningServiceRefusesToSignDueToSlashingCondition() {
+    final BeaconBlock block = dataStructureUtil.randomBeaconBlock(10);
+    client.when(request()).respond(response().withStatusCode(412));
```

Comment:
Can we add this as a constant in `HttpStatusCodes` and use that constant rather than hard coding the 412 number?

Reply:
👍

Reply:
done ... both in the test and external signer.